mono/tests/tailcall/split-fsharp.cpp

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Remove this file.

Reply:
This file is critical to maintenance of the tests. I have used it repeatedly as I edit the tests and it saved me a ton of time. I tried writing in Perl but gave up.

Reply:
Its imho we already have test generating files like tests/gen-runtime-invoke.cs.